jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -0,0 +1,137 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent none
+    environment {
+        BUILD_MANIFEST = "build-manifest.yml"
+        INTEG_TEST_JOB_NAME = 'integ-test-opensearch-dashboards'
+    }
+    parameters {
+        string(
+            name: 'TEST_MANIFEST',
+            description: 'Test manifest under the manifests folder, e.g. manifests/2.0.0/opensearch-dashboards-2.0.0-test.yml.',
```

Comment:
We are already adding manifest path as `manifests/` here: https://github.com/opensearch-project/opensearch-build/pull/1653/files#diff-3f7479e22eeeedaf481e3de253091fff48673c90624a9e1101a57924864d9b8eR149
I think the example should be corrected to `/2.0.0/opensearch-dashboards-2.0.0-test.yml`

Reply:
Correct but if triggering the integ-test job manually it will still need that manifests, for example: manifests/2.0.0/opensearch-dashboards-2.0.0-test.yml will work but /2.0.0/opensearch-dashboards-2.0.0-test.yml will show `No such file or directory: '/var/jenkins/workspace/integ-test-opensearch-dashboards/1.2.0/opensearch-dashboards-1.2.0-test.yml`

Reply:
So are the args different for manual vs automated run of this file?

Reply:
Should be the same, 

Manually we can include the `manifests` but at the time of automation we have to append `manifests` to the test run arg: 
https://github.com/opensearch-project/opensearch-build/pull/1653/files#diff-3f7479e22eeeedaf481e3de253091fff48673c90624a9e1101a57924864d9b8eR149.

OpenSearch is doing the same but I understand the confusion. When building, we call groovy scripts adding `manifests`. So instead of passing it to the test job. We can just pass `/2.0.0/opensearch-dashboards-2.0.0-test.yml` and then in the test script we can append `manifests` like how we do while building. For example [this](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch-dashboards/distribution-build.jenkinsfile#L46).

Reply:
I see but when we run the above example file we pass manifest as `2.0.0/test.yml`
![image](https://user-images.githubusercontent.com/61760125/156433427-14408122-f87b-4ace-b890-5eb5ea7c80c7.png)
Was wondering if we can keep the manual and automated way as the same?

Reply:
Correct that is for the build pipeline, for the test pipeline this is how it looks:

![Screen Shot 2022-03-02 at 12 15 06 PM](https://user-images.githubusercontent.com/12060704/156442365-27ca4d5f-8172-4bb9-ad1d-507e891c94c5.png)

I went ahead and updated the OpenSearch Dashboards and OpenSearch to match what the build pipeline is doing.